samples/wifi/scan/prj.conf

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF700X=y
+
+# System settings
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_NANO=n
+
+# Networking
+CONFIG_NETWORKING=y
+CONFIG_NET_LOG=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_UDP=y
+CONFIG_NET_TCP=y
+CONFIG_NET_DHCPV4=y
+CONFIG_DNS_RESOLVER=y
```

Comment:
We can remove these, but check if driver gets build.

Reply:
Tried by removing these configs, Scan results event is not processing as expected.

Reply:
We can leave it as is then, but these are not strictly needed, some are needed for net_mgmt but they should be auto-selected.